components/support/images/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <dimen name="mozac_support_images_size_default">102dp</dimen>
```

Comment:
Where did this number come from?

Reply:
We moved the existing number here.

Reply:
I see from `mozac_browser_thumbnails_size_default`. Shouldn't we delete it from there then?

Reply:
Missed this comment. Will follow up

Reply:
Actually `mozac_browser_thumbnails_size_default` was already deleted.